### DISCUSSION TITLE
Enable CSV Import for the CSV I exported from LInkAce

### DISCUSSION THREAD

**Discussion by Pureway** [Kovah/LinkAce#534] [Support] [answered]
So, I installed LInkAce on my other PC. I thought I could import my previously exported LinkAce CSV to my new LinkAce installation and to my surprise it only accepts HTML. So, it would be great if it's made to import CSV.

Also, the HTML doesn't import Lists and it only imports Tags. So, it'd be great if it can import Lists as well.

Thank you all,

**Comment by Kovah** [comment, marked as answer]
Hi, neither the export nor the import feature are meant to be used as a backup, or to move data between two installations, only to import or export from other applications.
If you have a fresh copy of LinkAce on your PC, please use the backup feature and restore the complete database with it.

https://www.linkace.org/docs/v1/configuration/application-backups/

If you are using the standard MySQL database, you should be able to import the database backup with the following command (not tested yet):
```
docker exec -i linkace_db_1 sh -c 'exec mysql -uroot -p"$MARIADB_ROOT_PASSWORD"' < /path/to/your/database-backups/mysql-linkace.sql
```

**Reply by Pureway** [reply]
Thanks for the clarification
